### PR TITLE
[10.0] [MIG] product_categ_image (superseed)

### PR DESCRIPTION
**Pull request by FFernandez-PlanetaTIC**
This superseeds #343

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Review by LRovira-PlanetaTIC — APPROVED** [review]

File: product_categ_image/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+{
+    'name': 'Product Category Image',
+    'version': '10.0.1.0.0',
+    'category': 'Sales',
+    'license': 'AGPL-3',
+    'summary': 'Add image on product category',
+    'author': 'Akretion,Odoo Community Association (OCA)',
+    'website': 'http://www.akretion.com',
```
Comment (bodedra) [inline product_categ_image/__manifest__.py]: Please change it to `https://github.com/OCA/product-attribute`

**Review by bodedra — CHANGES_REQUESTED** [review]

File: product_categ_image/__manifest__.py (hunk 1 above)
Comment (FFernandez-PlanetaTIC) [reply product_categ_image/__manifest__.py]: Done

**Comment by bodedra** [comment]
IMO these module will overlapping functionality of `product_default_image`

Would you please check these.

(product_default_image) https://github.com/OCA/product-attribute/blob/10.0/product_default_image/models/product_category.py#L21

(product_categ_image) https://github.com/OCA/product-attribute/pull/391/files#diff-007c4a91e47f76c2a433cdc4552ce629R12

@pedrobaeza @LRovira-PlanetaTIC

**Comment by FFernandez-PlanetaTIC** [comment]
We can put a note in the README saying that it is incompatible with `product_default_image`.

@bodedra , @pedrobaeza , what do you think?

Regards

**Comment by pedrobaeza** [comment]
Yes, put that in the readme and you can declare the incompatibility in the manifest (it's not supported yet on 10.0, but it will on 12.0)

**Comment by FFernandez-PlanetaTIC** [comment]
How can I declare the incompatibility in the manifest? I can not find how to do it

File: product_categ_image/README.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+   :alt: License: AGPL-3
+
+======================
+Product Category Image
+======================
+
+This module adds an *image* field on Product Categories.
+
+Many e-commerce software have the ability to set an image on a product category, so we need to have this field in Odoo in order to sync with these e-commerce software.
+
```
Comment (bodedra) [inline product_categ_image/README.rst:12]:
```suggestion
Incompatible
============

This module is incompatible with product_default_image module.

```

**Comment by pedrobaeza** [comment]
Use the key `conflicts` with a list.

File: product_categ_image/README.rst (hunk 2 above)
Comment (FFernandez-PlanetaTIC) [reply product_categ_image/README.rst:12]: Thanks!

File: product_categ_image/README.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+   :alt: License: AGPL-3
+
+======================
+Product Category Image
+======================
+
+This module adds an *image* field on Product Categories.
+
+Many e-commerce software have the ability to set an image on a product category, so we need to have this field in Odoo in order to sync with these e-commerce software.
+Incompatible
```
Comment (bodedra) [inline product_categ_image/README.rst:13]: Please add one empty line before.

**Comment by bodedra** [comment]
@FFernandez-PlanetaTIC Could you squash your commits. Migration looks good. Good job.

File: product_categ_image/README.rst (hunk 3 above)
Comment (FFernandez-PlanetaTIC) [reply product_categ_image/README.rst:13]: Thanks

**Comment by FFernandez-PlanetaTIC** [comment]
@bodedra Done

**Review by bodedra — APPROVED** [review]
LGTM :+1:

**Pull request closed without merging**